common/persistence/sql/sqlTaskManager.go

```diff
@@ -528,6 +528,29 @@ func (m *sqlTaskManager) CompleteTasksLessThan(
 	return int(nRows), nil
 }
 
+// GetOrphanTasks gets tasks from the tasks table that belong to a task_list no longer present
+// in the task_lists table.
+func (m *sqlTaskManager) GetOrphanTasks(ctx context.Context, request *persistence.GetOrphanTasksRequest) (*persistence.GetOrphanTasksResponse, error) {
```

Comment:
Same here: Would you mind add a TODO comment here linking to issue that I just opened? #4064